src/sage/combinat/k_regular_sequence.py

```diff
@@ -1049,6 +1049,25 @@ def guess(self, f, n_max=100, max_dimension=10, sequence=None):
 
         A :class:`kRegularSequence`
 
+        ALGORITHM:
+
+        For the purposes of this description, the left vector valued sequence
+        associated with a regular sequence is consists of the left vector
```

Comment:
```suggestion
        associated with a regular sequence consists of the left vector
```

Reply:
Copied to https://github.com/sagemath/sage/pull/35682/files#r1207823514.